pytorch/inference/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -20,7 +20,6 @@ ENV LANG C.UTF-8
 ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
 ENV PATH /opt/conda/bin:$PATH
 ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
-ENV TEMP=/home/model-server/tmp
```

Comment:
Does it break anything or just not needed?

Reply:
the TEMP variable is not used in the dockerfile. removed as it's not needed